### DISCUSSION TITLE
show axis value at the end of the axis

### DISCUSSION THREAD

**Discussion by plunavat** [jfree/jfreechart#411] [Q&A]
Hello,

We are generating xy scatter chart. The maximum value of the axis should always end at the major axis with values. But sometimes the axis shows minor gridlines after major axis. How do I ensure that the domain and range axis end at the major axis with value shown

![image](https://github.com/user-attachments/assets/5603cade-db5a-474d-8d15-6f3a48ac56bd)

Regards,
Pranav

**Comment by trashgod** [comment]
As far as I know, axis tick marks coincide with plot gridlines. Assuming you have specified a zero axis offset, you might try making the tick marks invisible:
```
plot.getRangeAxis().setTickMarksVisible(false);
plot.getRangeAxis().setMinorTickMarksVisible(false);
```
